### DISCUSSION TITLE
Extensions not visible to IDE?

### DISCUSSION THREAD

**Discussion by prescience-data** [supabase/supabase#3000] [Questions] [answered]
I've configured the Supabase Postgres connection which seems to resolve most things, however the Postgres extensions I've installed don't appear to be visible to the IDE (even when prefixing with an `extensions.<EXT_NAME>` qualifier).

I've set my resolution scope to `Everything`, and it _can_ see the `extensions` schema (syntax highlighter lights up when referencing it), but apparently none of the extensions within?

Is this due to some security configuration I need to adjust on the server (or access via a different user/method) / some other trick to make it show up?

I wasn't exactly sure how to word this so apologies if already answered!

![image](https://user-images.githubusercontent.com/65471523/131270815-b5f9b2d1-aa20-4170-ae67-66798db4451d.png)

**Comment by soedirgo** [comment, marked as answer]
I can access `moddatetime` (after enabling it in Database > Extensions) for a new project from psql and DBeaver:

<img width="1241" alt="Screen Shot 2021-08-30 at 3 30 44 PM" src="https://user-images.githubusercontent.com/31685197/131302611-f6c51739-e501-4977-b455-d8182274a7ad.png">

So this looks like something to do with the IDE.

**Comment by prescience-data** [comment]
FYI for anyone searching this in future:
If you are using a JetBrains IDE, make sure you've selected the schemas you want the IDE to have visibility on:

![image](https://user-images.githubusercontent.com/65471523/132148378-7c80a03c-eca5-4fd8-9039-9daf5bac6cf4.png)

**Reply by unhyif** [reply]
@prescience-data Hi, this is a late question tho, where is this feature? is this a paid plugin? I'm using Webstorm
